### PR TITLE
Feature/135/implement create for art

### PR DESCRIPTION
**Pull request by kiyanovskaa**

### CODE REVIEW

File: Streetcode/Streetcode.BLL/MediatR/Media/Art/Create/CreateArtHandler.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            _repositoryWrapper = repositoryWrapper;
+            _logger = logger;
+        }
+
+        public async Task<Result<ArtCreateUpdateDTO>> Handle(CreateArtCommand request, CancellationToken cancellationToken)
+        {
+            var newArt = _mapper.Map<DAL.Entities.Media.Images.Art>(request.Art);
+            if (newArt is null)
+            {
+                const string errorMsg = "Cannot convert ArtCreateUpdateDTO to Art";
```
Comment (metrondir) [inline Streetcode/Streetcode.BLL/MediatR/Media/Art/Create/CreateArtHandler.cs:27]: use error message from resource

**Review by Shavronskyy — CHANGES_REQUESTED** [review]
remove file program.cs or comment line 35

**Review by Shavronskyy — APPROVED** [review]
LGTM

**Review by metrondir — APPROVED** [review]
lgtm

**Pull request merged**